week-1-practise/level-2/4-currency-conversion-2.js

```diff
@@ -7,10 +7,17 @@
   Find a way to add 1% to all currency conversions (think about the DRY principle)
 */
 
-function convertToUSD(price) {}
-function convertToBRL(price) {}
+//As all currency conversions raised by 1%
+//function extra(price){
+  //return 1.01* price}
 
-/* DO NOT EDIT BELOW THIS LINE */
+function convertToUSD(price) {
+  return (1.3* extra(price)).toFixed(2);
```

Comment:
Much clearer 👌 

(It looks like you commented out your `extra` function though)

Reply:
Also see previous comments